pallets/msa/src/rpc/src/lib.rs

```diff
@@ -53,6 +53,10 @@ pub trait MsaApi<BlockHash, AccountId> {
 		delegator_msa_id: DelegatorId,
 		provider_msa_id: ProviderId,
 	) -> RpcResult<Option<Vec<SchemaId>>>;
+
+	/// Retrieve a list of MSA keys for a given MSA ID
+	#[method(name = "msa_getMsaKeys")]
+	fn get_msa_keys(&self, msa_id: MessageSourceId) -> RpcResult<(BlockNumber, Vec<AccountId>)>;
```

Comment:
return type for RPC is a tuple (BlockNumber, Vec<AccountId>) is ok? or needs beautification

Reply:
it's fine for a spike. We don't need to make it perfect yet